ts/features/barcode/screens/BarcodeScanScreen.tsx

```diff
@@ -155,12 +154,7 @@ const BarcodeScanScreen = () => {
         });
         break;
       case "ITW_REMOTE":
-        navigation.navigate(ITW_REMOTE_ROUTES.MAIN, {
-          screen: ITW_REMOTE_ROUTES.CLAIMS_DISCLOSURE,
-          params: {
-            itwRemoteRequestPayload: barcode.itwRemoteRequestPayload
-          }
-        });
```

Comment:
Since we are already extracting data from the url, we could pass the payload direclty to the screen. What do you think?